code/__DEFINES/jobs.dm

```diff
@@ -47,6 +48,7 @@
 #define EXECUTIVE_OFFICER "Executive Officer" //Currently disabled.
 #define FIELD_COMMANDER "Field Commander"
 #define STAFF_OFFICER "Staff Officer"
+#define TADPOLE_OFFICER "Tadpole Officer"
```

Comment:
This name is not new player friendly (new player thinks tadpole is an animal), rename it to something like transport pilot
Might want to rename rest of code for consistency as well